src/supernova.c

```diff
@@ -49,177 +49,250 @@ char outfile[256];
 char outfile_smeared[256];
 char bgfile[256];
 
+int my_chan_num = 0;
+
+
```

Comment:
This variable is not actually used anywhere; the only other occurrence is commented out